### PR TITLE
Completed Submit Handler for Recommend Docket Switch View

### PR DESCRIPTION
**Pull request by jcq**
Connects #15420

### Description
This fleshes out the onSubmit callback in `RecommendDocketSwitchContainer` to properly format the task instructions and perform the POST to the API to create the `DocketSwitchRulingTask`.

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] `DocketSwitchRulingTask` gets properly created upon submission of valid form data

### CODE REVIEW

File: client/app/queue/docketSwitch/recommendDocketSwitch/recommendDocketSwitchSlice.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
```
Comment (leikkisa) [inline client/app/queue/docketSwitch/recommendDocketSwitch/recommendDocketSwitchSlice.js:1]: Ooh nice.

**Review by leikkisa — COMMENTED** [review]
Hi JC - this looks great overall! I think you might need to add something to interpret the formatting when it's showing up in the timeline. I'll add a screenshot to the PR.  I don't specifically remember this being a problem with MTV...?

Another piece of feedback I have is that "Send to Judge" is not an intuitive task action name for the judge to be creating their ruling.

**Comment by leikkisa** [comment]
Screen Shot 2020-10-29 at 9 53 32 AM

**Comment by jcq** [comment]
> Another piece of feedback I have is that "Send to Judge" is not an intuitive task action name for the judge to be creating their ruling.

This looks like something got missed in review for the new task actions; fixing that!

**Comment by jcq** [comment]
> Hi JC - this looks great overall! I think you might need to add something to interpret the formatting when it's showing up in the timeline. I'll add a screenshot to the PR. I don't specifically remember this being a problem with MTV...?

Oof, you're right. I somehow overlooked this portion of the formatting of task instructions. The current behavior for the instructions on the case timeline is to simply run `StringUtil.nl2br` on them, which will cause it to honor new lines, but otherwise no other formatting. 

My intention was to parse these instructions as markdown, which we can easily do on the form page when a user is acting upon the task. Here, however... hmm.

**Review by leikkisa — APPROVED** [review]
Approving this - and we can break out the Case Timeline formatting into a separate ticket.  It'd be easy to sneak in the task name update though 😉 .

**Comment by jcq** [comment]
> Approving this - and we can break out the Case Timeline formatting into a separate ticket. It'd be easy to sneak in the task name update though 😉 .

Updated the task action labels, and created #15531 as a follow-up ticket

**Pull request merged**